--- /tmp/tmp.lslsMJhInv/clean_env/lib/python3.11/site-packages/gym/utils/passive_env_checker.py	2026-03-15 12:12:25.705793917 +0100
+++ .venv/lib/python3.11/site-packages/gym/utils/passive_env_checker.py	2026-03-15 12:03:26.859111815 +0100
@@ -230,7 +230,7 @@
         )
         obs, reward, done, info = result
 
-        if not isinstance(done, (bool, np.bool8)):
+        if not isinstance(done, (bool, np.bool_)):
             logger.warn(
                 f"Expects `done` signal to be a boolean, actual type: {type(done)}"
             )
@@ -238,11 +238,11 @@
         obs, reward, terminated, truncated, info = result
 
         # np.bool is actual python bool not np boolean type, therefore bool_ or bool8
-        if not isinstance(terminated, (bool, np.bool8)):
+        if not isinstance(terminated, (bool, np.bool_)):
             logger.warn(
                 f"Expects `terminated` signal to be a boolean, actual type: {type(terminated)}"
             )
-        if not isinstance(truncated, (bool, np.bool8)):
+        if not isinstance(truncated, (bool, np.bool_)):
             logger.warn(
                 f"Expects `truncated` signal to be a boolean, actual type: {type(truncated)}"
             )
