--- /tmp/tmp.xKpsKGMK9f/orig/gym_super_mario_bros/smb_env.py	2022-06-21 03:28:46.000000000 +0200
+++ .venv/lib/python3.11/site-packages/gym_super_mario_bros/smb_env.py	2026-03-15 12:02:02.009950697 +0100
@@ -95,22 +95,22 @@
     @property
     def _level(self):
         """Return the level of the game."""
-        return self.ram[0x075f] * 4 + self.ram[0x075c]
+        return int(self.ram[0x075f]) * 4 + int(self.ram[0x075c])
 
     @property
     def _world(self):
         """Return the current world (1 to 8)."""
-        return self.ram[0x075f] + 1
+        return int(self.ram[0x075f]) + 1
 
     @property
     def _stage(self):
         """Return the current stage (1 to 4)."""
-        return self.ram[0x075c] + 1
+        return int(self.ram[0x075c]) + 1
 
     @property
     def _area(self):
         """Return the current area number (1 to 5)."""
-        return self.ram[0x0760] + 1
+        return int(self.ram[0x0760]) + 1
 
     @property
     def _score(self):
@@ -139,7 +139,7 @@
     def _x_position(self):
         """Return the current horizontal position."""
         # add the current page 0x6d to the current x
-        return self.ram[0x6d] * 0x100 + self.ram[0x86]
+        return int(self.ram[0x6d]) * 0x100 + int(self.ram[0x86])
 
     @property
     def _left_x_position(self):
